plugins/airbrake/package.json

```diff
@@ -25,6 +25,7 @@
   "dependencies": {
     "@backstage/catalog-model": "^0.10.0",
     "@backstage/core-components": "^0.8.9",
+    "@backstage/core-app-api": "^0.5.3",
```

Comment:
Hm! This one should not be needed at all!

Reply:
Removed :)

Reply:
Ah, but then I'd prefer that you actually fake it instead - just implement the interface with its single method

```ts
const fake = {
  getBaseUrl(t: string) {
    return `http://localhost:7007/api/${encodeURIComponent(t)}`
  }
} 
```

or so

Reply:
oh no, hang on, you are initializing this mock from somewhere right - you may have been able to grab the actual discoveryApiRef at that point and injecting it right?

Reply:
@freben done!

Reply:
Yeah I am directly using it here: https://github.com/backstage/backstage/pull/9605/files#diff-80249c8b6f20164606712f880c923987b3cf4bd7af18a8f4a216b5d6fb7a9f61R70

And I can make do by just implementing the type.